src/pages/home/report/ReportActionsView.js

```diff
@@ -140,8 +140,7 @@ class ReportActionsView extends React.Component {
             if (!ReportActionComposeFocusManager.isFocused()) {
                 return;
             }
-
-            this.scrollToListBottom();
+            ReportScrollManager.scrollToBottom();
```

Comment:
Ok, Why aren't we calling `scrollToBottomAndUpdateLastRead` here?

Reply:
This is explained in the issue description:

>Remove updateLastReadActionID on keyboardDidShow ( not instructed. I have tested it on android: updateLastReadActionID is being called each time user open keyboard. if user open a room and he is on a room, updateLastReadActionID is called automatically, whether he is viewing old message (scroll at the top) or not. So, when scroll at the top and user open keyboard updateLastReadActionID is unnecessary. )

I think the assumption is that we are already updating the last read when the report opens and so it is unnecessary to do it again when the keyboard opens. Feels fine to me but maybe there's some reason why we are setting it when the keyboard opens? I would guess the original intention is to just scroll to the bottom. But the excess calls to the API are because those two behaviors were stuck together before.

Reply:
I don't remember why it was added so it seems fine. But let's use add proper tests so that QA can test the unread/read behavior properly. 

All scenarios should be tested when messages are marked read.
1. When we switch between reports. 
2. When we resize the window on Desktop from mobile to Web.
3. etc...

Reply:
CC @K4tsuki  See :arrow_up: